ui/shared/components/panel/variants/VariantUtils.jsx

```diff
@@ -60,3 +60,29 @@ export const ProteinSequence = React.memo(({ hgvs }) => <Sequence color="black"
 ProteinSequence.propTypes = {
   hgvs: PropTypes.string.isRequired,
 }
+
+const variantIntervalOverlap = (variant, padding, noGenomeVersion) => (interval) => {
+  const { pos, end, genomeVersion, liftedOverPos } = variant
+  const variantPos = (genomeVersion === interval.genomeVersion) || noGenomeVersion ? pos : liftedOverPos
```

Comment:
For RNAseq outliers, no genome version is provided in the data. Should we always use `pos` rather than `leftedOverPos`?

Reply:
yes

Reply:
I'm not sure the `noGenomeVersion` is really a necessary argument here - couldn't you just do this condition based on whether or not `interval.genomeVersion` is defined?

Reply:
The existing code allows `interval.genomeVersion` to be undefined. In this case, it will use the `liftedOverPos` rather than `pos`. Is it a desired behavior? If not, we could change its behavior to use `pos` when `interval.genomeVersion` is undefined.

Reply:
its probably better to swap the logic around to check if `variant.liftedOverGenomeVersion === interval.genomeVersion` and use that to determine if `liftedOverPos` should be used